README.md

```diff
@@ -33,7 +33,7 @@ doc/rocketmq-cpp_manaual_zh.docx
 
 ## Build and Install ##
 
-### Linux platform ###
+### Linux and Mac platform ###

```

Comment:
It would be better to replace Mac with macOS

Reply:
change to  Linux and Mac OS,   and delete platform